extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -76,40 +76,47 @@ public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 	}
 
 	public static function onArticleContentOnDiff( $diffEngine, \OutputPage $output ) {
-		global $wgTitle, $wgCityId;
+		global $wgTitle, $wgCityId, $wgRequest;
 
 		if ( $wgTitle->inNamespace( NS_MEDIAWIKI )
 			&& $wgTitle->isJsPage()
 			&& $wgTitle->userCan( 'content-review' )
 		) {
-			\Wikia::addAssetsToOutput( 'content_review_diff_page_js' );
-			\JSMessages::enqueuePackage( 'ContentReviewDiffPage', \JSMessages::EXTERNAL );
-
-			$output->prependHTML(
-				\Xml::element( 'button',
-					[
-						'class' => 'content-review-diff-button',
-						'data-wiki-id' => ( $wgCityId ),
-						'data-page-id' => \Title::newFromText( $wgTitle->getArticleID() ),
-						'data-status' => ReviewModel::CONTENT_REVIEW_STATUS_REJECTED
-					],
-					wfMessage( 'content-review-diff-reject' )->plain()
-				)
-			);
-
-			$output->prependHTML(
-				\Xml::element( 'button',
-					[
-						'class' => 'content-review-diff-button',
-						'data-wiki-id' => ( $wgCityId ),
-						'data-page-id' => \Title::newFromText( $wgTitle->getArticleID() ),
-						'data-status' => ReviewModel::CONTENT_REVIEW_STATUS_APPROVED
-					],
-					wfMessage( 'content-review-diff-approve' )->plain()
-				)
-			);
+			$reviewModel = new ReviewModel();
+			$pageId = \Title::newFromText( $wgTitle->getArticleID() );
+			$reviewData = $reviewModel->getReviewedContent( $wgCityId, $pageId, ReviewModel::CONTENT_REVIEW_STATUS_IN_REVIEW );
+			$diff = $wgRequest->getInt( 'diff' );
+			if ( !empty( $reviewData ) && (int)$reviewData['revision_id'] === $diff ) {
```

Comment:
You do not need the `(int)` type casting here, it is covered by the `->getInt()` method.